client/src/pages/_app.tsx

```diff
@@ -22,17 +28,24 @@ const fontWorkSans = Work_Sans({
   variable: "--font-worksans",
 });
 
+const fontFugazOne = Fugaz_One({
+  weight: "400",
+  subsets: ["latin"],
+  variable: "--font-fugazone",
+});
+
 const queryClient = new QueryClient();
 
 export default function App({ Component, pageProps }: AppProps) {
   return (
     <div
-      className={`${fontMontserrat.variable} ${fontPlusJakartaSans.variable} ${fontWorkSans.variable}`}
+      className={`${fontMontserrat.variable} ${fontPlusJakartaSans.variable} ${fontWorkSans.variable} ${fontFugazOne.variable}`}
     >
       <QueryClientProvider client={queryClient}>
         <ReactQueryDevtools initialIsOpen={false} />
-        <div className="w-full">
-          <main className="mx-auto flex min-h-screen max-w-7xl flex-col items-center gap-4 p-24">
+        <Navbar />
+        <div className="w-full pt-16">
+          <main>
```

Comment:
This is fine but ideally we don't have too many unnecessary `div` tags. You can add the classes to `main`
 ```suggestion
          <main className="w-full pt-16">
          ...
          </main>
```